Akkadu/rsi-widgets
# Documentation of RSI API (GET PUSH PUT DELETE) Event

## Motivation and Context
In order to work with RSI SDK Cliente, you need a way to create and handle your events without going through `akkadu.com`. Originally we will use the SDK Manager, but we have decided to provide too in first place a API (which the manager as well as the customer could use directly).

~This is the `template` or `draft` of how this API documentation will be showed and explained in the docs.  Which will be updated in this PR or further for each new API integrated.~

**Update**
Include docs for GET `/events/languages`
Include docs for PUSH PUT DELETE GET `/events`
Include PUT DELETE `/events/interpreters`
Include a Tutorial Section.


## How To Test
Check the [new section](https://deploy-preview-25--rsi-akkadu-documentation.netlify.app/interpreter-api/index.html)


## Checklist



加油！
